chapter10/voice-werewolf/werewolf/simulator.py

```
@@ -0,0 +1,441 @@
+                response_format="mp3",
+            )
+            path.write_bytes(response.content)
+            request_id = getattr(response, "_request_id", None)
+            provider = "OpenAI Audio API"
+        else:
+            path = self.out_dir / f"{stem}.wav"
+            voice = os.getenv("SIMULATOR_ESPEAK_VOICE", "en-us")
+            model = f"espeak-{voice}"
+            with tempfile.TemporaryDirectory(prefix="werewolf-simulator-tts-") as directory:
+                source = Path(directory) / "speech.wav"
+                subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)